DEVELOPERNOTES.md

```diff
@@ -0,0 +1,71 @@
+## Heads-up
+
+The library requires hhvm 3.30 and will therefore use hhvm-autoload 1.
+This version ought to be run with HHVM, not PHP.
+Failing to do so will result in this error:
+```
+Fatal error: Uncaught Error: Class undefined: Facebook\AutoloadMap\IncludedRoots in /.../quizlet/hammock/vendor/hhvm/hhvm-autoload/bin/hh-autoload:86
+```
+
+## Required patches to composer packages
```

Comment:
Are these patches only required for HHVM 3?

Reply:
You can't install this package on HHVM 4.yy, since the composer.json requires that you use 3.30.zz. I'll clarify as soon as that restriction is lifted.